src/App.js

```diff
@@ -68,60 +62,40 @@ function App() {
 
   const [searchValue, setSearchValue] = useState("");
   const [searchQuery, setSearchQuery] = useState({});
-  useEffect(() => setSearchValue(""), [waveIndex]);
 
   const visibleMentorIds = useMemo(
     () =>
       searchValue.trim().length === 0
-        ? mentorIds[waveIndex]
-        : fieldSearch(searchQuery, waveIndex),
-    [searchQuery, searchValue, waveIndex]
+        ? mentorIds
+        : fieldSearch(searchQuery),
+    [searchQuery, searchValue]
   );
 
   return (
     <div className="container">
       <Header />
 
-      <SearchBar
+      {/* <SearchBar
         value={searchValue}
         waveIndex={waveIndex}
         onSearchChange={setSearchValue}
         onSearchSelect={setSearchQuery}
-      />
-
-      <Tabs
-        className="tabs-container"
-        selectedIndex={convertIndex(waveIndex)}
-        onSelect={activateTab}
-      >
-        <TabList>
-          {/* Sort waves in descending order. */}
-          {waves
-            .slice()
-            .reverse()
-            .map(({ name }, i) => (
-              <Tab key={i}>{name}</Tab>
-            ))}
-        </TabList>
-
-        {waves.map((_, i) => (
-          <TabPanel key={i}>
-            <div className="card-container">
-              {visibleMentorIds.map((mentorId) => (
-                <ProfileCard
-                  key={mentorId}
-                  mentor={mentors[mentorId]}
-                  onReadMore={() => activateModal(mentorId)}
-                />
-              ))}
-            </div>
-          </TabPanel>
-        ))}
-      </Tabs>
+      /> */}
+
+      <div className="tabs-container">
+        <div className="card-container">
+          {visibleMentorIds.map((mentorId) => (
+            <ProfileCard
+              mentor={mentors.filter(mentor => mentor.name === mentorId)[0]}
```

Comment:
This line seems rather strange to me, as `mentorId` is supposed to be some sort of slug that corresponds to the mentor name, e.g. "John Doe" would correspond to mentorId "JohnDoe". I see that you made changes to make `mentorId` simply the mentor's name, but I'm not certain if the hash stuff will still work for special names, e.g. names with apostrophes/brackets in them. Also, this filtering solution is not ideal.

Ideally in a future PR, `mentors` should be changed into an object that has keys being the `mentorId` (which as explained above will be a slug derived from the mentor name; see https://github.com/AdvisorySG/mentorship-page/blob/master/scripts/process_mentors.py#L80 for the Python code I used to generate the slug, which should be extended to remove all non-alphanumeric letters and ensure all IDs are unique).